vmdb/db/migrate/20150401090146_add_systemd_and_openstack_related_columns_to_system_services.rb

```diff
@@ -0,0 +1,7 @@
+class AddSystemdAndOpenstackRelatedColumnsToSystemServices < ActiveRecord::Migration
+  def change
+    add_column :system_services, :systemd_load, :string
+    add_column :system_services, :systemd_active, :string
+    add_column :system_services, :systemd_sub, :string
```

Comment:
@roliveri, @movitto 

How do we proceed with this?  Does @ladas have to change this to use the serialized `dependencies` column?  Or, can he use the new `systemd_`-specific columns?

Time is critical with this change because it involves DB Schema changes and we're coming up on a code freeze.